playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml

```diff
@@ -4,8 +4,9 @@
   serial: 1
   vars:
     sap_hypervisor_node_platform: redhat_ocp_virt
-
-  tasks:
-    - name: Include Role
-      ansible.builtin.include_role:
-        name: sap_hypervisor_node_preconfigure
+    sap_hypervisor_node_kubeconfig: "{{ lookup( 'ansible.builtin.env', 'KUBECONFIG') }}"
+  environment:
+    KUBECONFIG: "{{ sap_hypervisor_node_kubeconfig }}"
+    K8S_AUTH_KUBECONFIG: "{{ sap_hypervisor_node_kubeconfig }}"
```

Comment:
I am no k8s expert, just double checking K8S_AUTH_KUBECONFIG should have the same value as KUBECONFIG?

Reply:
Some modules / tasks use KUBECONFIG, some K8S_AUTH_KUBECONFIG. Thats why they _are_ set identically. The value is read from the environment from the calling environment and has to be set specifically for ansible.